package.json

```diff
@@ -116,28 +116,20 @@
     "eslint-plugin-jest": "^23.0.4",
     "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-react-hooks": "^2.2.0",
-    "expect": "^1.16.0",
     "extract-text-webpack-plugin": "^4.0.0-beta.0",
+    "fake-indexeddb": "^3.0.0",
     "file-loader": "^5.0.2",
     "fs-extra": "^0.30.0",
     "html-webpack-plugin": "^3.2.0",
+    "jest": "^24.9.0",
+    "jest-environment-jsdom-thirteen": "^1.0.1",
```

Comment:
any reason we're using JSDOM13 when the https://www.npmjs.com/package/jest-environment-jsdom-thirteen page basically tells you to use 14?

Reply:
The Google told me to. Will test with fourteen and see what happens (probably nothing breaking)

Reply:
14 looks fine